docs/messaging/messaging-spans.md

```diff
@@ -236,20 +237,23 @@ interpret linked traces without the need for additional semantic hints.
 
 #### Producer spans
 
-"Publish" spans SHOULD be created for operations of providing messages for
-sending or publishing to an intermediary. A single "Publish" span can account
-for a single message, or for multiple messages (in the case of providing
-messages in batches). "Create" spans MAY be created. A single "Create" span
-SHOULD account only for a single message. "Create" spans SHOULD either be
-children or links of the related "Publish" span.
+"Create" spans MAY be created when a message is created or passed to the client
+library or other component responsible for publishing.  A single "Create" span
+SHOULD account only for a single message.  "Publish" spans SHOULD be created
```

Comment:
```suggestion
SHOULD account only for a single message.  

"Publish" spans SHOULD be created
```

What if we break the line, so each "block" refers to one "thing"? I think that will improve readability.

Reply:
If have no strong opinion, we can do that if people think it increases readibility.

The logic I was following with the paragraphs:
* Paragraph 1: What spans to create
* Paragraph 2: What context to inject
* Paragraph 3: What links to create